core/create/src/index.ts

```diff
@@ -147,17 +154,19 @@ async function mainPrompt() {
   )
 }
 
-function confirmationPrompt() {
+function confirmationPrompt(deleteConfig: boolean, addEslintConfig: boolean) {
   return prompt({
     name: 'confirm',
     type: 'confirm',
-    message: (_prev, values) => {
```

Comment:
yes, it wasn't working previously (I don't remember exactly whether it was a {} or undefined, but `values` was one of those). I suspect that's because here we're calling prompt separately instead of chaining this confirm object with the rest of the objects [here](https://github.com/Financial-Times/dotcom-tool-kit/blob/main/core/create/src/index.ts#L92)

Reply:
yeah, I refactored them to be separate at some point but must have missed that. thanks for spotting it!